java/flight/flight-sql/src/test/java/org/apache/arrow/flight/TestFlightSql.java

```diff
@@ -180,6 +180,14 @@ public void testGetTablesSchema() {
     collector.checkThat(info.getSchema(), is(FlightSqlProducer.Schemas.GET_TABLES_SCHEMA));
   }
 
+  @Test
+  public void testGetTablesSchemaExcludeSchema() {
+    final FlightInfo info = sqlClient.getTables(null, null, null, null, false);
+    collector.checkThat(info.getSchema(), is(FlightSqlProducer.Schemas.GET_TABLES_SCHEMA_NO_SCHEMA));
+  }
+
+
+
```

Comment:
nit: no need for so many newlines
```suggestion
  public void testGetTablesSchemaExcludeSchema() {
    final FlightInfo info = sqlClient.getTables(null, null, null, null, false);
    collector.checkThat(info.getSchema(), is(FlightSqlProducer.Schemas.GET_TABLES_SCHEMA_NO_SCHEMA));
  }

```